src/c4/Timer.cc

```diff
@@ -347,8 +346,8 @@ void Timer::printline_mean(std::ostream &out, unsigned const p,
   double s = sum_system_cpu(), s2 = s * s;
```

Comment:
@KineticTheory I know you didn't touch this line or the next, but isn't this (multiple initializations on a line) frowned upon in our style guide?

Reply:
See #795